hackIDE/static/hackIDE/js/custom.js

```diff
@@ -411,6 +411,17 @@ $(document).ready(function(){
 							$(".error-message").html(response.run_status.stderr);
 						}
 					}
+					// To handle case when hackerearth api doesn't return a valid response.
```

Comment:
The response which consists the "Error code: 1200" as its compile status, is a success as the API informs us about its inability to compile the code sent to it. Also when this response is returned by the API we see "Error code: 1200" in the output box, which would not be the case if the ajax request failed. (If it did fail we would get a "Server couldn't complete request. Please try again!" message)